test/integration/minishift.go

```diff
@@ -28,6 +28,11 @@ import (
 
 var lastCommandOutput CommandOutput
 
+const (
+	ConsoleURL   = "console --url"
```

Comment:
Not sure whether I am a fan of using constants here. Are you planning to re-use these?